netket/models/rbm.py

```diff
@@ -229,7 +229,7 @@ def setup(self):
 
     @nn.compact
     def __call__(self, x_in):
-        x = nknn.DenseSymm(
+        x = nknn.symmetric_linear.DenseSymmMatrix(
```

Comment:
We cannot break existing code...

Can't you do `nknn. DenseSymm(... mode="matrix")` ?